CHANGELOG.md

```diff
@@ -79,6 +79,8 @@
   the project is changed from the last saved snapshot and lighter when the
   snapshot matches the current project state.
 - [Added shortcut to interrupt the program][3967]
+- [Added dropdown for node argument value suggestions.][4013] Currently it
```

Comment:
This should be understandable by the user (user != power user). This sentence is really hard to parse. Can we make it a little bit more elaborate and easier to understand, please?